packages/admin-ui/components/pages/setup/dataSources/BigQueryProperties.tsx

```diff
@@ -82,6 +82,32 @@ export default function BigQueryProperties() {
       >
         <Input placeholder="Project-123" />
       </Form.Item>
+      <Form.Item
+        label="Dataset"
+        required
+        name="dataset"
+        rules={[
+          {
+            required: true,
+            message: ERROR_TEXTS.CONNECTION.DATASET.REQUIRED,
+          },
+        ]}
+      >
+        <Input placeholder="natality" />
```

Comment:
What is the placeholder means?

Reply:
It's a public dataset name. Not sure should be add tip string or not.

right now, I think... we don't want to confuse users, so I will remove the placeholder later.